examples/tests/microbit_v2/README.md

```diff
@@ -0,0 +1,59 @@
+Micro:bit v2 Test App

```

Comment:
This entire README is great! Very helpful.

Reply:
Thanks!